mixer/README.md

```diff
@@ -1,5 +1,22 @@
 # Mixer
 
+## Mixer v2
```

Comment:
I think we decided not to call it Mixer v2. We've been referring to it as "telemetry v2" or "new extension architecture".

Reply:
great. will fix.

Reply:
BTW, the Mixer deprecation doc calls it "Extensions v2" so I used that.